lib/convection/model/template/resource/aws_efs_file_system.rb

```diff
@@ -0,0 +1,24 @@
+require_relative '../resource'
+
+module Convection
+  module Model
+    class Template
+      class Resource
+        # Represents an {https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-efs-filesystem.html
+        # Amazon EFS File System}
+        class EFSFileSystem < Resource
+          include Model::Mixin::Taggable
```

Comment:
Since the actual name is FileSystemTags, I wonder if it might be worth adding an alias for file_system_tag just to keep the naming consistent with what Amazon uses